core/psh/Makefile

```diff
@@ -4,9 +4,11 @@
 # Copyright 2018, 2019, 2020, 2021 Phoenix Systems
 #
 
-ifdef PSH_DEFUSRPWDHASH
+PSH_DEFUSRPWDHASH ?= ""
 CFLAGS += -DPSH_DEFUSRPWDHASH=\"$(PSH_DEFUSRPWDHASH)\"
-endif
+
+PSH_WILDCARDS ?= ""
```

Comment:
please document somewhere how this functionality should be used - either in `phoenix-rtos-doc` or - at least - write here comment about expected variable syntax

Reply:
Documentation added and waiting as PR in phoenix-rtos-doc/utils/psh.md